internal/models/reservation_model.go

```diff
@@ -52,6 +52,9 @@ type AWSReservation struct {
 
 	// The ID of the image from which the instance is created.
 	AMI string `db:"ami" json:"ami"`
+
+	// The ID of the the aws reservation which was created.
```

Comment:
It must have a ReservationID or else the instance wouldn't have been created
I mean: We first create the reservation, and only after the reservation was created successfully we will launch an aws instance and create a db record

Reply:
I think it make sense, we can't test every corner case, so unless there is a real worry it could cause some real trouble in the future, I'm ok not to have explicit test for that :+1:

Reply:
@adiabramovitch can you fix the nit? :)